### DISCUSSION TITLE
MP3 capital issue (versus mp3)

### DISCUSSION THREAD

**Discussion by regencape** [music-assistant/support#3216] [Q&A] [answered]
Capital issue not in the sense that it is very important, but in the sense of... capitals. Maybe I should issue a bug, but I'm a bit unsure whether I'm doing something wrong or missing a known issue. :) 

I have some media files on a local server that are not indexed in MA. I use MA as an add-on in Hassio. It all (more or less) works fine, but it seems that files with the extension MP3 (in capitals) are not recognized. Can't even find them through browse function within MA. Don't ask me why they have the extension MP3, but the local server doesn't want me to change it to mp3. It changes it back to capitals. When I copy the file to my desktop, rename it without the capitals and copy it back to the server, it is recognized by MA right away. Anyhow, hopefully somebody can reproduce this problem and confirm that the capitals are indeed the problem? Would this be a valid bugreport?

**Comment by OzGav** [comment]
This is noted. It will take some time to respond due to other priorities right now. Thanks for your patience.

**Reply by OzGav** [reply, marked as answer]
Fixed in 2.3.4 beta to follow
